packages/web-components/fast-foundation/src/progress-ring/progress-ring.template.ts

```diff
@@ -23,38 +31,25 @@ export function progressRingTemplate<T extends FASTProgressRing>(
             ${when(
                 x => typeof x.value === "number",
                 html<T>`
-                    <svg
-                        class="progress"
-                        part="progress"
-                        viewBox="0 0 16 16"
-                        slot="determinate"
+                    <span
+                        class="determinate"
+                        part="determinate"
+                        style="--percent-complete: ${x => x.percentComplete}"
                     >
-                        <circle
-                            class="background"
-                            part="background"
-                            cx="8px"
-                            cy="8px"
-                            r="7px"
-                        ></circle>
-                        <circle
-                            class="determinate"
-                            part="determinate"
-                            style="stroke-dasharray: ${x =>
-                                (progressSegments * x.percentComplete) /
-                                100}px ${progressSegments}px"
-                            cx="8px"
-                            cy="8px"
-                            r="7px"
-                        ></circle>
-                    </svg>
+                        <slot name="determinate">
+                            ${options.determinateIndicator ?? ""}
+                        </slot>
+                    </span>
                 `
             )}
             ${when(
                 x => typeof x.value !== "number",
                 html<T>`
-                    <slot name="indeterminate">
-                        ${options.indeterminateIndicator ?? ""}
-                    </slot>
+                    <span class="indeterminate" part="indeterminate">
+                        <slot name="indeterminate">
+                            ${options.indeterminateIndicator ?? ""}
+                        </span>
+                    </span>
```

Comment:
It looks like progress and progress ring are basically the same template now - is there a reason we need to maintain two in this case? Seems like where the deltas were previously are now primarily implementation details.

Reply:
Right, that was my intent. I think we don't need to maintain two, but in that model do we even need to keep two components or do we remove the "ring"? Should we take that as a followup?

Reply:
I think if these are the same we likely don't need both - you should be able to setup separate instances using the same class and templates which will reduce our bundle. If we want to remove as a follow-up, can we open an issue to track?

Reply:
Created #6362.